### PR TITLE
1.0.7 release

### PR DESCRIPTION
**Pull request by arcalinea**

### CODE REVIEW

File: doc/man/zcash-cli.1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASH-CLI "1" "February 2017" "zcash-cli v1.0.6" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH ZCASH-CLI "1" "March 2017" "zcash-cli v1.0.7" "User Commands"
 .SH NAME
-zcash-cli \- manual page for zcash-cli v1.0.6
+zcash-cli \- manual page for zcash-cli v1.0.7
 .SH DESCRIPTION
-Zcash RPC client version v1.0.6\-dirty
+Zcash RPC client version v1.0.7\-dirty
```
Comment (bitcartel) [inline doc/man/zcash-cli.1]: Remove -dirty? Shouldn't really have been in previous commit.

File: doc/man/zcash-tx.1 [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASH-TX "1" "February 2017" "zcash-tx v1.0.6" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH ZCASH-TX "1" "March 2017" "zcash-tx v1.0.7" "User Commands"
 .SH NAME
-zcash-tx \- manual page for zcash-tx v1.0.6
+zcash-tx \- manual page for zcash-tx v1.0.7
 .SH DESCRIPTION
-Zcash zcash\-tx utility version v1.0.6\-dirty
+Zcash zcash\-tx utility version v1.0.7\-dirty
```
Comment (bitcartel) [inline doc/man/zcash-tx.1]: Ditto,  remove dirty.

File: doc/man/zcashd.1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASHD "1" "February 2017" "zcashd v1.0.6" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH ZCASHD "1" "March 2017" "zcashd v1.0.7" "User Commands"
 .SH NAME
-zcashd \- manual page for zcashd v1.0.6
+zcashd \- manual page for zcashd v1.0.7
 .SH DESCRIPTION
-Zcash Daemon version v1.0.6\-dirty
+Zcash Daemon version v1.0.7\-dirty
```
Comment (bitcartel) [inline doc/man/zcashd.1]: Ditto remove dirty

**Comment by str4d** [comment]
- Release notes appear to include 1.0.6 changes too (e.g. ZMQ).
- Man pages have -dirty in the versions
  - In release notes, need to change version, commit, then build, then generate man pages.

File: doc/man/zcash-cli.1 (hunk 1 above)
Comment (arcalinea) [inline doc/man/zcash-cli.1]: Probably occurs because there were uncommitted changes? Perhaps we could just change release documentation

**Comment by str4d** [comment]
ACK

@zkbot r+

**Pull request merged**